lib/src/models/message.dart

```diff
@@ -39,7 +41,8 @@ class Message {
     ReplyMessage? replyMessage,
     this.reaction = '',
     this.messageType = MessageType.text,
-  }) : replyMessage = replyMessage ?? ReplyMessage();
+  })  : replyMessage = replyMessage ?? ReplyMessage(messageId: id),
+        key = GlobalKey();
```

Comment:
I guess this will override the value passed from constructor.

Reply:
We are not taking key from constructor. It is directly being assigned this way.